apps/prairielearn/src/lib/editors.ts

```diff
@@ -192,21 +192,35 @@ export function getUniqueNames({
  * `defaultValue` may be either a value to compare directly with `===`, or a function
  * that accepts a value and returns a boolean to indicate if it should be considered
  * a default value.
+ *
+ * You should set `isUIBoolean` to true if the property is a UI boolean that controls the visibility of another field.
+ * For example, `beforeDateEnabled` is a UI boolean that controls the visibility of the `beforeDate` field.
```

Comment:
Nit, it's not controlling the visibility, it's controlling whether or not the input is enabled.